org-roam-graph.el

```diff
@@ -190,6 +191,16 @@ If PREFIX, then the graph is generated but the viewer is not invoked."
 	        (call-process org-roam-graph-viewer nil 0 nil temp-graph)
         (view-file temp-graph)))))
 
+(defun org-roam-graph-show-connected-component (&optional prefix)
```

Comment:
This function probably needs a check for whether it's an org-roam file via `org-roam--org-roam-file-p`.

Reply:
In principle this should not be necessary, since non-roam files are not present in the database, consequently the query does not select any nodes. I tested it and the current behavior is an empty graph for non-roam files.
Should we still introduce an explicit check? And if yes: Keep the empty graph behavior or emit a warning?

Reply:
Yes, it doesn't error out. The point is to not generate the graph at all for a non-roam file, since it doesn't make sense to.

Reply:
Sounds good, I'll also take care of the rebase.